spk/salt-master/Makefile

```
@@ -91,6 +91,6 @@ endif
 salt-master_extra_install:
 	install -m 644 src/app/rsax931.py.patch $(STAGING_DIR)/share
 	install -m 644 src/app/master $(STAGING_DIR)/share
-	install -m 755 -d $(STAGING_DIR)/share/extensions
-	install -m 755 -d $(STAGING_DIR)/share/extensions/auth
-	install -m 644 src/app/synology.py $(STAGING_DIR)/share/extensions/auth/synology.py
+	install -m 755 -d $(STAGING_DIR)/var/extensions
+	install -m 755 -d $(STAGING_DIR)/var/extensions/auth
```

you can omit the first line, install -d creates the whole path (as you didn't create the var folder)
```suggestion
	install -m 755 -d $(STAGING_DIR)/var/extensions/auth
```